Fix accessibility reported by pa11y

senthil10 opened the pull request:
These are accessibility issues reported by pa11y. These are not raised by wave, none of this have any affect visually or functionally, but it is a better change code wise.

Issued addressed:

* Duplicate id in same page
* Missing `alt` tags
* Missing title for iframes

senthil10 commented on layouts/partials/navbar.html:
```diff
@@ -2,7 +2,7 @@
     <!-- Logo & Subtitle -->
     <div class="d-flex flex-column logo">
-      <a href="/"><img src="/img/logos/sll_dp_outline.svg" aria-label="SciLifeLab Data Platform" ></a>
+      <a href="/"><img src="/img/logos/sll_dp_outline.svg" alt="SLL DP Logo" aria-label="SciLifeLab Data Platform" ></a>
```
If there is an `aria-label` attribute exists, screen readers will use it, that is why `wave` did not complain about this.

Anyway will fix it while fixing other texts ^^

Ziip-dev replied:
Ah right, this takes precedence. No issue then :)

senthil10 commented:
@Ziip-dev I have fixed the text, note the `PlumX` is a tool not abbreviation, it is mentioned in the text above, so guess we can't do much there.

Also, I have fixed what was bugging me before, the redundant `img` tags, have fixed that in `css`

Ziip-dev reviewed (approved):
Perfect work! :ok_hand:
